toothpick-compiler/src/main/java/toothpick/compiler/common/ToothpickProcessor.java

```diff
@@ -312,8 +314,12 @@ private boolean isAnnotationPresent(TypeElement annotationTypeElement, String an
     return false;
   }
 
-  private boolean isSameType(TypeElement annotationTypeElement, String annotationTypeName) {
-    return typeUtils.isSameType(annotationTypeElement.asType(), elementUtils.getTypeElement(annotationTypeName).asType());
+  private boolean isSameType(TypeElement typeElement, String typeName) {
+    return typeUtils.isSameType(typeElement.asType(), elementUtils.getTypeElement(typeName).asType());
```

Comment:
could reuse the other isSameType?

return isSameType(typeElement.asType, typeName)